<launch>
    <node pkg="order_receiver" type="order_receiver_node" name="order_receiver" output="screen">
        <param name="/pub_topic_name" type="string" value="/tiny_car_order"/>
        <!-- <param name="/serial_device" type="string" value="/dev/ttyUSB0"/> -->
        <param name="/serial_device" type="string" value="/dev/pts/21"/>
        <param name="/loop_rate" type="int" value="1000"/>
        <param name="/baud_rate" type="int" value="115200"/>
    </node>
</launch>
